subgraph.template.yaml

```diff
@@ -213,6 +213,34 @@ dataSources:
           handler: handleSellZap
         - event: Swap(uint256,uint256,address)
           handler: handleSwapZap
+  - kind: ethereum
+    name: ZeroXMarketplaceZap
+    network: goerli
```

Comment:
should this be `network: {{network}}' instead?

Reply:
ty, have fixed